pkg/network/server.go

```diff
@@ -177,6 +177,7 @@ func (s *Server) Start(errChan chan error) {
 
 	s.tryStartConsensus()
 
+	go s.broadcastStaleTxLoop()
```

Comment:
Don't `go` here, it's just not needed any more.

Reply:
And there is no `Loop` inside anyway.

Reply:
Fixed.